### PR TITLE
Inheritance Modes Update

### PR DESCRIPTION
**Pull request by drio18**
This PR fixes a few small errors with current inheritance mode calculations for trios, specifically those associated with sex chromosomes. Here, we:

- add X-linked inheritance modes for female probands
- fix an erroneous X-linked inheritance mode from a father to a male proband (relegating it to a new low relevance label)
- change some capitalization/spellings of other labels
- update the extended descriptions/tests accordingly
- update inheritance mode facet order overrides for SNVs and SVs

Branch was deployed on cgaptest with both an SNV and SV VCF ingested.

Note that changes to the inheritance mode labels/rescue terms make previously ingested cases have inheritance mode facet oddities. For example, the switch from "Dominant (maternal)" to "Dominant (Maternal)" means both terms will appear in an old case, with the former facet dropping to the bottom of the facet list while the latter will have zero counts. Perhaps I should write an upgrader for this? Or manually patch these values? I may have to do the same for filtersets to keep them functioning as expected.

### CODE REVIEW

File: src/encoded/docs/extended_description_StructuralVariantSample_inheritance_mode.html [hunk 1]
```diff
@@ -6,14 +6,16 @@ <h5>
...
-<li> <b> X-linked dominant (Paternal)</b>: Father: 1, Mother: 0/0, Son: 1.
-<li> <b> Y-linked dominant</b>: Father: 1, Son: 1.
-<li> <b> Loss of Heterozygosity</b>: Father: 0/1, Mother: 0/0, Child: 1/1.
-<li> <b> Loss of Heterozygosity</b>: Father: 0/0, Mother: 0/1, Child: 1/1.
+<li> <b> Dominant (Paternal)</b>: Father: 0/1, Mother: 0/0, Child 0/1.
+<li> <b> Dominant (Maternal)</b>: Father: 0/0, Mother: 0/1, Child 0/1.
+<li> <b> Homozygous recessive</b>: Father: 0/1, Mother: 0/1, Child 1/1.
+<li> <b> X-linked recessive</b>: Father 1, Mother: 0/1, Daughter 1/1.
+<li> <b> X-linked dominant (Maternal)</b>: Father 0, Mother 0/1, Daughter 0/1.
+<li> <b> X-linked dominant (Paternal)</b>: Father 1, Mother 0/0, Daughter 0/1.
```
Comment (sbreiff) [inline src/encoded/docs/extended_description_StructuralVariantSample_inheritance_mode.html:14]: Could this also be Father 1, Mother 0/0, Son 1?

File: src/encoded/docs/extended_description_VariantSample_inheritance_mode.html [hunk 2]
```diff
@@ -8,17 +8,19 @@ <h5>Inheritance mode values are candidate labels based on novocaller, Cmp-Het ca
...
 <ul>
 <li> <b> Dominant (Paternal)</b>: Father: 0/1, Mother: 0/0, Child 0/1.
 <li> <b> Dominant (Maternal)</b>: Father: 0/0, Mother: 0/1, Child 0/1.
-<li> <b> Recessive</b>: Father: 0/1, Mother: 0/1, Child 1/1.
-<li> <b> X-linked Recessive</b>, <b>X-linked Dominant (Maternal)</b>: Father: 0, Mother: 0/1, Son 1.
-<li> <b> X-linked dominant (Paternal)</b>: Father: 1, Mother: 0/0, Son 1.
+<li> <b> Homozygous recessive</b>: Father: 0/1, Mother: 0/1, Child 1/1.
+<li> <b> X-linked recessive</b>: Father 1, Mother: 0/1, Daughter 1/1.
+<li> <b> X-linked dominant (Maternal)</b>: Father 0, Mother 0/1, Daughter 0/1.
+<li> <b> X-linked dominant (Paternal)</b>: Father 1, Mother 0/0, Daughter 0/1.
```
Comment (sbreiff) [inline src/encoded/docs/extended_description_VariantSample_inheritance_mode.html:22]: what about son?

File: src/encoded/docs/extended_description_StructuralVariantSample_inheritance_mode.html (hunk 1 above)
Comment (sbreiff) [reply src/encoded/docs/extended_description_StructuralVariantSample_inheritance_mode.html:14]: Actually I see now that this is given a low-relevance mode.

**Review by sbreiff — APPROVED** [review]
These look good to me. As far as an upgrader, I'd recommend against it in favor of just patching all the items.

**Review by willronchetti — APPROVED** [review]
Code looks good. As far as upgrader vs. patching I think an upgrader will be less work for you, but it is your call on how you want to handle it. Keep in mind that we may need to run this operation across envs (such as in MSA where we have already ingested a lot of variants).

**Pull request merged**